Rack/app/models/bookings_gear.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+class BookingsGear < ActiveRecord::Base
```

Comment:
This is a redundant model, you need it if you create `has_many through` relation. But if you use `has_and_belongs_to_many` relation, you must create only join table in the database, but without a model. 

Notice, you use `dependent: :destroy` in `belongs_to` relation, so if you destroy any `BookingsGear` record, you will destroy related `booking` and `gear` records too.

Reply:
understand